src/Meilisearch/Extensions/HttpExtensions.cs

```diff
@@ -81,13 +83,49 @@ internal static void AddDefaultUserAgent(this HttpClient client)
             client.DefaultRequestHeaders.Add("User-Agent", version.GetQualifiedVersion());
         }
 
-        private static StringContent PrepareJsonPayload<T>(T body, JsonSerializerOptions options = null)
+        private static StringContent PrepareJsonPayload<T>(T body, JsonSerializerOptions? options = null)
         {
             options = options ?? Constants.JsonSerializerOptionsWriteNulls;
             var payload = new StringContent(JsonSerializer.Serialize(body, options), Encoding.UTF8, "application/json");
             payload.Headers.ContentType.CharSet = string.Empty;
 
             return payload;
         }
+
+        private static Task<HttpResponseMessage> PatchAsync(this HttpClient client, string? requestUri, HttpContent content, CancellationToken cancellationToken)
+        {
+            var uri = string.IsNullOrEmpty(requestUri) ? null : new Uri(requestUri, UriKind.RelativeOrAbsolute);
+            return client.PatchAsync(uri, content, cancellationToken);
+        }
+
+        private static Task<HttpResponseMessage> PatchAsync(this HttpClient client, Uri? requestUri, HttpContent content, CancellationToken cancellationToken)
+        {
+            // HttpClient.PatchAsync is not available in .NET standard and NET462
+            var method = new HttpMethod("PATCH");
+            var request = new HttpRequestMessage(method, requestUri) { Content = content };
+            return client.SendAsync(request, cancellationToken);
+        }
+
+        internal static Task<HttpResponseMessage> PatchAsJsonAsync<TValue>(this HttpClient client, string? requestUri, TValue value, JsonSerializerOptions? options = null, CancellationToken cancellationToken = default)
+        {
+            if (client == null)
+            {
+                throw new ArgumentNullException(nameof(client));
+            }
+
+            var content = JsonContent.Create(value, mediaType: null, options);
+            return client.PatchAsync(requestUri, content, cancellationToken);
+        }
+
+        internal static Task<HttpResponseMessage> PatchAsJsonAsync<TValue>(this HttpClient client, Uri? requestUri, TValue value, JsonSerializerOptions? options = null, CancellationToken cancellationToken = default)
```

Comment:
Actually, I think you can opt for just the `string` one, this file is just a collection of methods that will add logic to the instance of a `HttpClient`. In other words this is not a `Interface` where you cannot leave without implementing all the methods that interface requires :) 

So it will be ok to remove the `Uri` one :)